cli/src/run_testenv.rs

```diff
@@ -278,111 +280,70 @@ pub async fn initialize_test_organization(
     Ok([alice_device, other_alice_device, bob_device])
 }
 
+pub enum TestenvConfig {
+    ServerAlreadyRunning(String),
```

Comment:
Why String ?

Reply:
That value come from `std::env::var` which directly return a string

Reply:
Isn't it better to be parsed before ?

Reply:
done